scripts/backup

```diff
@@ -53,12 +53,9 @@ ynh_backup "/usr/share/transmission"
 
 ynh_backup "/var/lib/transmission-daemon"
 
-backup_core_only=$(ynh_app_setting_get $app backup_core_only)
-# If backup_core_only have a value in settings.yml, do not backup the data directory
-if [ -z $backup_core_only ]
-then
-	CHECK_SIZE "/home/yunohost.transmission"
-	ynh_backup "/home/yunohost.transmission"
-else
-	echo "Data dir will not saved, because backup_core_only is set." >&2
-fi
+CHECK_SIZE "/home/yunohost.transmission"
+# The "--is big" parameter indicates that the file/directory requires "a lot of disk space"
+# It won't be backed up before 'upgrade' or while doing a 'backup' with:
+# "sudo BACKUP_CORE_ONLY=1 yunohost backup create --apps nextcloud"
+# See https://yunohost.org/#/backup
+ynh_backup "/home/yunohost.transmission" --is_big
```

Comment:
To use --is_big, you have to use getopts for all arguments.
So put --src_path= for the first argument.

Also, small typo before, transmission instead of nextcloud into the comment.

Reply:
getopts needs YNH 3.5 if I remember correctly?

Reply:
`--is_big` is a getopts argument.